lwreg/utils.py

```diff
@@ -251,7 +254,8 @@ def connect(config):
         _replace_placeholders = _replace_placeholders_pcts
     else:
         _replace_placeholders = _replace_placeholders_noop
-    _dbConnection = cn
+    if _lookupWithDefault(config, "cacheConnection"):
+        _dbConnection = cn
```

Comment:
The `configure_from_database()` function will open a cached connection when it is called.
It makes sense here to clear any existing cached connections if we disable caching.
```suggestion
    if _lookupWithDefault(config, "cacheConnection"):
        _dbConnection = cn
    else:
        _dbConnection = None
```

Reply:
I left the kwarg `cache_connection` in `configure_from_database()` to be able to prevent that from happening.